### ISSUE TITLE
The GUI rerun should support changing the initial conditions and time array.

### ISSUE THREAD

**Issue by moorepants** [pydy/pydy#206]
There is no way to change the initial conditions or time array from the GUI. These are important items to be able manipulate.

**Comment by oliverlee** [comment]
Keep in mind that the system may not be defined and just the trajectory is available, so these options won't always be useable.

**Comment by moorepants** [comment]
Yes, these would only be changeable if you are using the `System` class to generate a scene, much like the constants are used now. We need to make sure that if arbitrary trajectories are passed in it is aware.

**Comment by oliverlee** [comment]
The time parameters PR hasn't been merged.
